Update PBS entry with all my progress

arynyestos opened the pull request:
PR made at the request of @taxmeifyoucan to add some comments to keep improving the entry.

taxmeifyoucan commented on docs/wiki/research/PBS/pbs.md:
This is great but instead of ethresearch reference, you can use our ePBS page I linked in a comment before and make this section leaner

taxmeifyoucan reviewed (commented):
Thanks for the PR @arynyestos!

I left few comments but the content is great, we can replace the original doc with it. Look at the doc diff to make sure you are ok with replacing all of the old content, maybe you want to keep something. Otherwise it's good to go

arynyestos commented on docs/wiki/research/PBS/pbs.md:
```diff
@@ -1,44 +1,178 @@
+
+## PBS Block Creation (MEV-Boost)
+
+In today's Ethereum, most validators use the [MEV-Boost](https://github.com/flashbots/mev-boost) client (MEV now meaning Maximal Extractable Value), which allows them to source high-MEV blocks from a competitive builder marketplace, in a similar fashion as MEV-Geth did with transaction bundles in PoW Ethereum. This Ethereum client makes it possible for validators to create such bundles taking tra ...
```
I did mention it above, do you mean that you wouldn't mention it here?

arynyestos commented on docs/wiki/research/PBS/pbs.md:
```diff
@@ -1,44 +1,178 @@
+
+Although not directly ingrained in the Ethereum protocol, PBS is a very relevant topic, not only due to its many current benefits, but also because it stands to play a significant role in the future of Ethereum. As explained in the [ethereum.org](https://ethereum.org/en/roadmap/pbs/) entry for PBS, outsourcing block construction to specialized builders will probably become essential when full Dan ...
+
+Thus, Danksharding being one of the key milestones in Ethereum's roadmap, and the way the network shall scale to over 100,000 TPS, PBS stands out as an indispensable practice to keep Ethereum decentralized.
```
Hmm, could you mention a few so that I can add them?

arynyestos commented:
> Thanks for the PR @arynyestos!
> 
> I left few comments but the content is great, we can replace the original doc with it. Look at the doc diff to make sure you are ok with replacing all of the old content, maybe you want to keep something. Otherwise it's good to go

I just made a new commit addressing all your comments to the best of my ability. Let me know what you think and whether I should merge it as it is now or tweak anything else :-)

taxmeifyoucan commented:
Thanks for all the updates! I think it's ready to go.
